internal/web_server/web/views/pages/dashboard_pages/access_settings_page.templ

```diff
@@ -2,9 +2,28 @@ package dashboard_pages
 
 import (
 	"github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/layout_components"
+	"github.com/Molnes/Nyhetsjeger/internal/models/users/access_control"
+	"github.com/Molnes/Nyhetsjeger/internal/web_server/web/views/components/dashboard_components/access_settings_components"
 )
 
-templ AccessSettingsPage() {
+templ AccessSettingsPage(admins *[]access_control.UserAdmin) {
 	@layout_components.DashBoardLayout("Tilgangs innstillinger") {
+		<div class="flex flex-col gap-6 px-8 py-6 max-w-screen-lg mx-auto">
+			<section>
+				<h1
+					class="text-3xl font-bold text-gray-800"
+				>Tilgang administrering</h1>
+				<p>
+					Her kan du adminstrere hvem har tilgang til <b>Quiz Admin Panel</b>.
+					<br/>
+					Hust at desse personene kan publisere quizzer, redigere dem,
+					og har tilgang til brukerinformasjon.
+					<br/>
+					Kun ogranisasjonens administratorer har tilgang til denne siden.
```

Comment:
Would prefer if you created new paragraphs instead of using `br`.

Also slightly change the text. My suggestion for text:
## Administrer tilgang
Her kan du administrere hvem som har tilgang til **Quiz Admin Panel**.
Disse brukerne kan lage, redigere, publisere og slette quizer, og har tilgang til brukernes informasjon.
Kun en organisasjons administrator har tilgang til nåværende siden.

Reply:
60058ea2da1475e734caa7edffccdd9d436492a6